frontend/integration-tests/views/cluster-settings.view.ts

```diff
@@ -16,3 +16,6 @@ export const clusterOperatorResourceLink = $('[data-test-id="console"]');
 export const globalConfigResourceLink = $('[data-test-id="Console"]');
 export const clusterResourceDetailsTitle = $('[data-test-id="resource-title"]');
 export const globalConfigDetailsTitle = $('[data-test-id="api-explorer-resource-title"]');
+export const globalConfigDetailsTitleIsLoaded = browser.wait(
+  until.presenceOf(globalConfigDetailsTitle),
+);
```

Comment:
Doesn't this need to be a function?

```suggestion
export const globalConfigDetailsTitleIsLoaded = () => browser.wait(
  until.presenceOf(globalConfigDetailsTitle),
);
```

Reply:
arrg..didn't push latest